### DISCUSSION TITLE
Use main or releases

### DISCUSSION THREAD

**Discussion by wesselhuising** [danny-avila/LibreChat#3519] [General]
Hi all,

Thank you for all the effort that is being put into this project. I am wondering, I see a lot of traction happening around the main branch but the last release Is over 1.5 months ago. Is that still used for broadcasting stable releases or should we pull from the main branch?

Cheers,

Wessel

**Comment by danny-avila** [comment]
> the last release Is over 1.5 months ago. Is that still used for broadcasting stable releases or should we pull from the main branch?

This is pretty standard for the repo. Unless something was announced, there's no reason why we would stop cutting stable releases.

There were a lot of optimization changes that I've been waiting for the dust to settle on for this particular release. I'm going to cut a release next week.

**Reply by wesselhuising** [reply]
Thank you for the quick response. Sounds great and I'll just wait for the next releases to come. Thanks again.

**Reply by ollieanwyll** [reply]
Thank you @danny-avila!

**Comment by danny-avila** [comment]
Just an update here, we are doing our best to cut a release today, as we have been doing some last minute polishing of the features added since v0.7.3

Thanks for your patience!
